src/main/java/org/fundacionjala/at15/katas/pokerhands/vladimir/StraightFlush.java

```diff
@@ -0,0 +1,47 @@
+package org.fundacionjala.at15.katas.pokerhands.vladimir;
+
+public class StraightFlush extends PokerHands {
```

Comment:
Your subclasses for `PokerHands` look good and you did a very nice job on separating the logic to identify the type of each group of cards.

I'd suggest to double check repeated code on those subclasses and see if you can create separated objects and/or methods that can set reusable code, this way you prevent having repeated code

Reply:
Yes i think about that too, i will do that, thanks Jose